.github/workflows/ci.yaml

```diff
@@ -108,42 +108,42 @@ jobs:
         with:
           node-version: "14"
 
-      - name: Fetch dependencies from cache
-        id: cache-yarn
-        uses: actions/cache@v2
-        with:
-          path: "**/node_modules"
-          key: yarn-build-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            yarn-build-
+      # - name: Fetch dependencies from cache
```

Comment:
Hmm strange this seems like it should work. :thinking: Maybe there was a change to the postinstall but not to the yarn.lock so the updated install was not ran? If so maybe we can fix this by also adding the postinstall scripts to the key hash.

It would be nice if we could leave all deps in the root instead of a separate vender directory but I imagine that could wreak havoc on certain parts of the build process.